recipes/linux/prefpicker.sh

```diff
@@ -17,16 +17,17 @@ case "${1-install}" in
   install)
     sys-embed \
       ca-certificates \
-      p7zip-full \
+      git \
```

Comment:
Is there a reason this isn't under `apt-install-auto` ? It shouldn't be a runtime dependency of prefpicker.